blog-service/2024-05-31-dashboards.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Multiple Scheduled Reports (Dashboards)
+image: https://help.sumologic.com/img/sumo-square.png
+keywords:
+  - search
+  - scheduled_reports
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
```

Comment:
@JV0812, you'll need to change the date on the release note, of course. Kevin Keech said in his [presentation](https://sites.google.com/sumologic.com/pd-global-home/products/explore-new-products#h.sek1vyg6k6rx) that it will release "sometime in the next two weeks".